xbmc/platform/linux/input/LIRC.h

```diff
@@ -20,62 +20,18 @@
 
 #pragma once
 
-#include <string>
-#include <atomic>
-
-#include "input/remote/IRemoteControl.h"
 #include "threads/Thread.h"
-#include "threads/Event.h"
+#include "input/InputManager.h"
```

Comment:
Can you move the IInput interface to a new file (maybe in an interfaces/ subdir) to avoid including this header here?

Reply:
I am not happy with this either and it was one of the last things I did for this PR. Lirc is now pretty self-managed and does not need InputManager. InputManager is not a good container for Lirc but I had no better idea. Suggestions welcome.